.ci/actions/main-sync/action.yml

```diff
@@ -24,9 +24,15 @@ inputs:
     default: false
 
 runs:
-  if: github.repository_owner == 'kiegroup'
   using: 'composite'
   steps:
+    - name: Terminate if the repository owner is not kiegroup
+      if: github.repository_owner != 'kiegroup'
+      shell: bash
+      run: |
+        echo "Teminating the workflow: repository owner is not kiegroup"
+        exit 1
```

Comment:
Just out of curiosity: do you know if `exit 1` will throw an error when the GHA tries to execute in the user's fork or it will just exit quietly? Just thinking if it throws an error the user would be receive emails about failures.

Reply:
Yes @rgdoliveira, `exit 1` terminates with an error. I will change to `exit 0` to terminate without errors and avoid emails to the user.

Reply:
@rgdoliveira the issue with `exit 0` is that it doesn't terminate and the workflow continues to the next step.
I tested it here: https://github.com/fantonangeli/incubator-kie-kogito-examples/actions/runs/8965363083
If we want to avoid to throw an error, I can only think about adding: 
`if: github.repository_owner != 'kiegroup'` 
to each step. In this case, I will use a variable to store the group name. WDYT?

Reply:
I see, maybe another option is that you add this conditional to the workflows (in each repo) that calls this one, wdyt? The bad part is that you would need to add this conditional to all other kogito/drools repos.

Reply:
@rgdoliveira I agree and together with this fix I'm opening the other kogito/drools repos PRs.
Now I updated and tested this last modification.